src/index.html

```diff
@@ -10,23 +10,47 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="search search--big"
+    >
+      <label
+        for="search__input--big"
+        class="search__label search__label--big"
+      >
+        <input
+          class="search__input search__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
The `input` element is missing the `name` attribute. The `name` attribute is used to reference form data after a form is submitted.